projects/shared/src/lib/directive/link-permissions.directive.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2022 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { AfterViewInit, Directive, ElementRef, Input, Renderer2 } from '@angular/core';
+import { PermissionsService } from '../service/permissions.service';
+
+@Directive({
+  // eslint-disable-next-line @angular-eslint/directive-selector
+  selector: '[linkPermissions]',
+})
+export class LinkPermissionsDirective implements AfterViewInit {
+
+  /** permissions */
+  private perms: string[] = [];
+
+  /** operator */
+  private andOperator: boolean = false;
```

Comment:
always the same ;-) And now the joke (in French) :
 
"Tu sais pourquoi Mickey Mousse ? parce que Mario Brosse !"